### PR TITLE
Set default values for sinker stuff.

### PR DESCRIPTION
**Pull request by spxtr**
/area prow

### CODE REVIEW

File: prow/cluster/starter.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ data:
   config: |
     prowjob_namespace: default
     pod_namespace: default
-    sinker:
+    sinker:0.18
```
Comment (krzyzacy) [inline prow/cluster/starter.yaml]: do we need sinker version here?

**Comment by krzyzacy** [comment]
small question, otherwise
/lgtm

File: prow/cluster/starter.yaml (hunk 1 above)
Comment (spxtr) [reply prow/cluster/starter.yaml]: Hah, fixed the regex in bump.sh.

**Comment by spxtr** [comment]
Also updated default namespaces to "default."

**Pull request merged**